src/shared/ethtool-util.c

```diff
@@ -453,11 +453,13 @@ static int get_glinksettings(int fd, struct ifreq *ifr, struct ethtool_link_uset
         if (ecmd.req.link_mode_masks_nwords <= 0 || ecmd.req.cmd != ETHTOOL_GLINKSETTINGS)
                 return -EOPNOTSUPP;
 
-        u = new0(struct ethtool_link_usettings , 1);
+        u = new(struct ethtool_link_usettings , 1);
```

Comment:
not that it matters, but there's an extra space before the `,` that is weird

Reply:
Fixed.